draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2208,6 +2234,44 @@ Steps:
 11. return (x, y)
 ~~~
 
+## Choosing a mapping function {#choosing-mapping}
+
+This section gives brief guidelines on choosing a mapping function
+for a given elliptic curve.
+Note that the suites given in {{suites}} are recommended mappings
+for the respective curves.
+
+If the target elliptic curve is a supersingular curve supported by either the
```

Comment:
Hmmm.... I'm not sure I understand how this suggestion would work.

Specifically, I think going from least specific (Weierstrass) to most specific (supersingular) would be confusing, and would lead people to make incorrect decisions. The reason for the current ordering is that it present essentially a simple decision tree: the most specific cases should come *first*, and then other cases "fall through."

Reply:
I am fine with going from specific to general. I suggested to reverse the order to follow the order of the sections.

Reply:
Ah, got it. Yeah, I tried to do that for the same reason and it got really messy...